exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -71,15 +72,26 @@ func NewPrwExporter(namespace string, endpoint string, client *http.Client, exte
 	}
 
 	return &PrwExporter{
-		namespace:      namespace,
-		externalLabels: sanitizedLabels,
-		endpointURL:    endpointURL,
-		client:         client,
-		wg:             new(sync.WaitGroup),
-		closeChan:      make(chan struct{}),
+		namespace:        namespace,
+		externalLabels:   sanitizedLabels,
+		endpointURL:      endpointURL,
+		client:           client,
+		wg:               new(sync.WaitGroup),
+		closeChan:        make(chan struct{}),
+		maxBatchByteSize: defaultMaxBatchByteSize,
 	}, nil
 }
 
+// SetMaxBatchByteSize sets the maximum uncompressed size to send in a single HTTP request.
+// This should be called before using the exporter.
+func (prwe *PrwExporter) SetMaxBatchByteSize(maxBatchByteSize int) {
```

Comment:
Why are we not passing `maxBatchByteSize` as a parameter to `NewPrwExporter` like we do for everything else? The special treatment and introduction of a setter for maxBatchByteSize is not clear.